ballerina/tests/data_binding_publish_tests.bal

```diff
@@ -617,6 +636,637 @@ service object {
         log:printInfo("Error Received: " + err.message());
         onErrorReceived = true;
         onErrorMessage = err.message();
+    }
+};
+
+@test:Config {
+    dependsOn: [testDataBindingPublishError],
+    groups: ["nats-basic"]
+}
+public function testDataBindingStringPayloadPublish() {
+    string messageToSend = "Testing Consumer Service With String Data Binding.";
+    Client? newClient = clientObj;
+    if newClient is Client {
+        Listener? sub = listenerObj;
+        if sub is Listener {
+            checkpanic sub.attach(consumerServiceStringPayloadPublish);
+            checkpanic sub.'start();
+            checkpanic newClient->publishMessage({ content: messageToSend, subject: DATA_BINDING_PUBLISH_SUBJECT });
+            int timeoutInSeconds = 120;
+            // Test fails in 2 minutes if it is failed to receive the message
+            while timeoutInSeconds > 0 {
+                if receivedStringValuePublish !is "" {
+                    string receivedMessage = receivedStringValuePublish;
+                    test:assertEquals(receivedMessage, messageToSend, msg = "Message received does not match.");
```

Comment:
```suggestion
                    test:assertEquals(receivedStringValuePublish, messageToSend, msg = "Message received does not match.");
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-nats/pull/427/commits/433577ffabea99c86a5ec34828b469bce521fc85